mod-source-record-storage-server/src/main/java/org/folio/verticle/MarcIndexersVersionDeletionVerticle.java

```diff
@@ -25,36 +35,87 @@ public class MarcIndexersVersionDeletionVerticle extends AbstractVerticle {
 
   private static final Logger LOGGER = LogManager.getLogger();
 
-  private RecordDao recordDao;
-  private TenantDataProvider tenantDataProvider;
+  private static final ZoneId ZONE_ID = ZoneId.systemDefault();
+
+  private final RecordDao recordDao;
+  private final TenantDataProvider tenantDataProvider;
+  private List<LocalTime> scheduleTimes;
+
+  @Value("${srs.marcIndexers.delete.time:01:00}")
+  private String deleteTimes;
+
+  @Value("${srs.marcIndexers.delete.oneTimeLimit:100000}")
+  private Integer oneTimeLimit;
```

Comment:
I think we should rename `oneTimeLimit` to `dirtyBatchSize`.

Reply:
Completed. Thank you